vars/failingTimeout.groovy

```diff
@@ -13,10 +13,21 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
-def call(Map conf, Closure body) {
+def call(Map args, Closure body) {
     // Modified from https://support.cloudbees.com/hc/en-us/articles/226554067/comments/360000870712
+    boolean activity = false
+    String unit = "MINUTES"
+
+    if (args.containsKey('activity')) {
+        activity = args.activity
+    }
+
+    if (args.containsKey('unit')) {
+        unit = args.unit
+    }
+
     try {
-        timeout(conf) {
+        timeout(time: args.time, activity: activity, unit: unit) {
```

Comment:
Where did you find the activity argument? Based on https://jenkins.io/doc/book/pipeline/syntax/ I don't see it

Reply:
It's here https://jenkins.io/doc/pipeline/steps/workflow-basic-steps/#timeout-enforce-time-limit